frontend/components/CellInput.js

```diff
@@ -833,13 +833,11 @@ const InputContextMenu = ({ on_delete, cell_id, run_cell, running_disabled, any_
                       ${running_disabled ? html`<span class="enable_cell ctx_icon" />` : html`<span class="disable_cell ctx_icon" />`}
                       ${running_disabled ? html`<b>Enable cell</b>` : html`Disable cell`}
                   </li>
-                  ${any_logs
```

Comment:
I'd like to keep the `any_logs` check so that we only show the option to hide/show if there are logs. Because if you have never seen logs, it's hard to understand what the button does.

Reply:
Makes sense! that makes the change mostly frontend since logs have to still be appended!